### PR TITLE
Update all resources

### PR DESCRIPTION
**Pull request by irmatov**
* add PK everywhere
* rename resource_id -> id

### CODE REVIEW

File: resources/postgresql_servers.go [hunk 1]
```diff
@@ -203,64 +204,16 @@ func PostgresqlServers() *schema.Table {
...
 				Columns: []schema.Column{
 					{
-						Name:        "server_id",
+						Name:        "server_cq_id",
 						Description: "Unique ID of azure_postgresql_servers table (FK)",
 						Type:        schema.TypeUUID,
 						Resolver:    schema.ParentIdResolver,
 					},
 					{
-						Name:        "resource_id",
```
Comment (irmatov) [inline resources/postgresql_servers.go:214]: Looks like a lot of fields were deleted. They were not: there was a duplicate relation that I have dropped, so it looks a bit not apparent in diff here.

**Comment by irmatov** [comment]
Please note that it can't be merged "as is" because it uses "replace" in go.mod to pull necessary fixes. So it requires updating after new -sdk release.

**Review by zagronitay — APPROVED** [review]
LGTM

**Review by roneli — APPROVED** [review]
LGTM

**Pull request merged**